models/public/hifi-gan/accuracy-check.yml

```diff
@@ -0,0 +1,27 @@
+models:
+  - name: lj_ft_t2_v1
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        adapter:
+          type: multi_output_regression
+          outputs:
+            - audio
+        allow_reshape_input: True
+        inputs:
+          - name: 'mel'
+            type: INPUT
+            layout: NHWC
+    datasets:
+      - name: HiFi-GAN
```

Comment:
Please move description of dataset conversion to [dataset_definitions.yml](https://github.com/openvinotoolkit/open_model_zoo/blob/master/data/dataset_definitions.yml)